crates/transaction-pool/src/test_utils/mock.rs

```diff
@@ -23,9 +23,11 @@ use reth_primitives::{
     BlobTransactionValidationError, PooledTransactionsElementEcRecovered, Signature, Transaction,
     TransactionSigned, TransactionSignedEcRecovered, TxType,
 };
-
 use std::{ops::Range, sync::Arc, time::Instant, vec::IntoIter};
 
+#[cfg(any(test, feature = "arbitrary"))]
+use reth_primitives_traits::HeuristicSize;
```

Comment:
if the `use reth_primitives_traits as _;` is required since this is feature gated, I'd try to instead make primitives-traits an optional dep, including it by default as a dev-dependency, and when `arbitrary` is enabled